2.2 Message Layout.md

```diff
@@ -150,9 +150,9 @@ A message is syntactically valid if:
 Messages that do not pass the Semantic Validation *shall be* discarded. Only semantically valid messages shall be allowed to continue in the data flow.
 
 A message is semantically valid if:
-1. The Message PoW Hash contains at least the number of leading 0 defined as required by the Rate Control module (see [Section 4.5 - Rate Control](./4.5%20Rate%20Control.md).
+1. The Message PoW Hash contains at least the number of leading 0 defined as required by the Rate Control module (see [Section 4.5 - Rate Control](./4.5RateControl).
```

Comment:
the name does not align with the actual file name. does it still interpret it correctly? or are you gonna change the file names after?

Reply:
Oh, good point. I'm actually not sure, but this shouldn't work. Although the workflow should check for broken links. Let me check

Reply:
Ok, it works because a slug is used 😅 
slug: 4.5RateControl

Reply:
On top of the markdown file of 4.5